src/components/Link/Link.vue

```diff
@@ -4,14 +4,21 @@
     :[linkProp]="to"
     :target="target"
     :rel="target === '_blank' ? 'noopener noreferrer' : ''"
+    :aria-disabled="disabled"
     :class="[
       {'underline text-primary-900' : underline},
-      {'primary py-3 px-6 bg-primary-500 text-white active:bg-primary-700 disabled:bg-white-300': primary},
-      {'secondary py-3 px-6 bg-white-200 border border-primary-500 text-primary-500 active:text-primary-700 active:border-primary-700 disabled:border-gray-100': secondary},
-      {'alert py-3 px-6 bg-white hover:bg-lemon-300 border rounded border-lemon-700 text-lemon-900 hover:text-lemon-900 focus:ring-lemon-700 disabled:border-gray-100': alert},
-      {'flex justify-center items-center rounded disabled:text-gray-500 focus:outline-none focus:ring-2 focus:ring-primary-100 focus:border-transparent': primary || secondary},
+      {'primary py-3 px-6 bg-primary-500 text-white active:bg-primary-700': primary},
+      {'secondary py-3 px-6 bg-white-200 border border-primary-500 text-primary-500 active:text-primary-700 active:border-primary-700': secondary},
+      {'tertiary py-3 px-6 bg-white border border-gray-100 text-gray-500 active:border-gray-300': tertiary},
+      {'alert py-3 px-6 bg-white hover:bg-lemon-300 border rounded border-lemon-700 text-lemon-900 hover:text-lemon-900 focus:ring-lemon-700': alert},
+      {'flex justify-center items-center rounded focus:outline-none focus:ring-2 focus:ring-primary-100 focus:border-transparent': primary || secondary || tertiary},
       {'hover:text-white': primary},
-      {'!px-3 !py-2': small}
+      {'!px-3 !py-2': small},
+      {'disabled pointer-events-none !text-gray-500': disabled},
```

Comment:
As mentioned in the PR, since `<a>` tags do not have a `disabled` attribute, I had to instead add a class here based on the prop. For the same reason, I also had to add the below conditional classes based on the prop, because `disabled:bg-white` (etc) don't work.